app/views/users/index.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: 'shared/user', collection: @users %>
```

Comment:
- [ ] Please make sure that the User Controller `#index` action renders all users to its corresponding Views index page (`app/views/users/index.html.erb`). Why don't you name your views files as `.html.erb`? Currently have this error while rending your user index page so make sure that you fix this issue :point_down:  and ensure that all your view are rendering correctly. 
  
  ![image](https://github.com/tsohleDev/weblog/assets/92303891/6c6d4e60-9a2a-478f-9915-612cd39bc30b)